src/main/java/net/minecraftforge/gradle/common/BasePlugin.java

```diff
@@ -368,7 +368,7 @@ public boolean isSatisfiedBy(Dependency element)
         }), null);
         if (fgDep == null) {
             // This shouldn't happen, unless people are doing really wonky stuff
-            throw new StopExecutionException("Missing FG dep in buildscript classpath");
```

Comment:
This fails during the tests. I'm assuming it's because "ForgeGradle" is not in the buildscript dependencies during the tests.